app/locale/de-DE.coffee

```diff
@@ -607,63 +607,66 @@ module.exports = nativeDescription: "Deutsch (Deutschland)", englishDescription:
     editor_config_behaviors_description: "Vervollständigt automatisch Klammern und Anführungszeichen."
 
   about:
-#    main_title: "If you want to learn to program, you need to write (a lot of) code."
-#    main_description: "At CodeCombat, our job is to make sure you're doing that with a smile on your face."
-#    mission_link: "Mission"
-#    team_link: "Team"
-#    community_link: "Community"
-#    story_link: "Story"
-#    careers_link: "Careers"
-#    press_link: "Press"
-#    mission_title: "Our mission: make programming accessible to every student on Earth."
-#    mission_description_1: "<strong>Programming is magic</strong>. It's the ability to create things from pure imagination. We started CodeCombat to give learners the feeling of wizardly power at their fingertips by using <strong>typed code</strong>." # {change}
-#    mission_description_2: "As it turns out, that enables them to learn faster too. WAY faster. It's like having a conversation instead of reading a manual. We want to bring that conversation to every school and to <strong>every student</strong>, because everyone should have the chance to learn the magic of programming." # {change}
-#    team_title: "Meet the CodeCombat team"
-#    team_values: "We value open and respectful dialog, where the best idea wins. Our decisions are grounded in customer research and our process is focused on delivering tangible results for them. Everyone is hands-on, from our CEO to our Github contributors, because we value growth and learning in our team."
-    nick_title: "Programmierer" # {change}
+    main_title: "Wenn du das Programmieren erlernen willst, musst du (viel) Code schreiben."
+    main_description: "Bei CodeCombat ist es unser Job das du das mit einem Lächeln im Gesicht tust."
+    mission_link: "Mission"
+    team_link: "Team"
+    community_link: "Community"
+    story_link: "Geschichte"
+    careers_link: "Karrieren"
+    press_link: "Presse"
+    mission_title: "Unsere Mission: Programmieren jedem Schüler auf der Erde zugänglich zu machen."
+    mission_description_1: "<strong>Programmieren ist Magie</strong>. Es ist die Fähigkeit Dinge aus purer Vorstellungskraft zu erschaffen.
+    Wir starteten CodeCombat um Lernen dieses Gefühl der Zauberkraft in ihren Fingerspitzen zu geben während sie <strong>getippten Code</strong> benutzen." # {change}
+    mission_description_2: "Wie sich herausstellt, können sie dadurch auch schneller lernen. Sehr viel schneller. 
+    Es ist mehr wie eine Unterhaltung als das Lesen einer Bedienungsanleitung. Wir wollen diese Unterhaltung in jede Schule zu
+    <strong>jedem Schüler</strong> bringen, weil jeder die Chance haben sollte die Magie des Programmierens zu erlernen." # {change}
```

Comment:
Ah, these two need to be on one line, since the line breaks here won't play well with our i18n tooling. Can you update to condense them into long, single lines?